README.md

```diff
@@ -1,22 +1,19 @@
 homebrew-font
 =============
 
-# Install fonts
+# Installing Ricty fonts
 
-Compile Ricty Font
-
-* Install
-   * Recommend
+Recommended installation by tapping the repository
 
 ```
-$ brew tap sanemat/font
+$ brew tap sanemat/homebrew-font
```

Comment:
I prefer `brew tap sanemat/font`, because many blog posts include [this](https://www.google.co.jp/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=%22sanemat%2Ffont%22). Would you keep `brew tap sanemat/font`?